pkgs/applications/science/logic/potassco/clingcon.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, cmake, bison, re2c }:
+
+stdenv.mkDerivation rec {
+  pname = "clingcon";
+  version = "3.3.0";
+  name = "${pname}-${version}";
```

Comment:
This attribute is created automatically if `pname` and `version` are used, no need for it :).